assets/scss/variables.scss

```diff
@@ -1,12 +1,12 @@
 //Tried to make a list of cohesive colors
-$primary-color: rgb(56, 215, 56);
+$primary-color: rgb(82, 209, 82);
```

Comment:
I think it should be 56,215,56 to match the rest of the green color. There already is enough contrast between them.